templates/data/kyma-integration-data.yaml

```
@@ -651,6 +668,22 @@ templates:
                     - "vm_job_template_k3d"
                     - "vm_job_labels_template"
                     - "extra_refs_kyma-local"
+              - jobConfig:
+                  name: "post-main-kyma-integration-k3d-telemetry"
+                  labels:
+                    preset-build-pr: "true"
```

postsubmit jobs are running against commits on the main branch
```suggestion
                    preset-build-main: "true"
```